database_functions.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+// Function to get categories from the database
+function getCategoriesFromDatabase()
```

Comment:
@gazalguptaa can we change the file name to a specific one? maybe `categories_query.php` or `category_functions.php` just so it's not too genericc

Reply:
Actually, I think this is a good idea - infact if we can, we should move all DB executions in a single file so it's easy for us to prove all the kind of queries we're doing. Super helpful for us to showcase the work for grades, since the major component is the DB queries
@gazalguptaa @jasmehr what do you think?

Reply:
It will also allow us to re-use queries tbh across different pages when needed. I have seen some MVC designs use this paradigm, but it's debatable. Happy to discuss more on this

Reply:
Resolving conversation, I believe this is fixed post in-person discussion